src/index.ts

```diff
@@ -201,11 +210,8 @@ const typescript: PluginImpl<RPT2Options> = (options) =>
 
 				if (output.emitSkipped)
 				{
-					noErrors = false;
```

Comment:
> The flag is only used to print a visible warning at the end of a cycle in watch mode (in case there is a lot of output that hides errors)

Yea I figured that out and that makes sense to me too 👍 

> I think this should have `noErrors = false` unconditionally, since we don't expect emit to ever be skipped.

This might have been an oversight on my behalf actually, since I didn't mention anything about this in the PR/commit message 🤔 
That being said, `emitSkipped` _should_ always have a diagnostic issue. The few times it hasn't (e.g. #338) has been due to a bug, so I'm not sure that there's any semantic difference in it being conditional or unconditional -- the condition is an invariant in this case.

Reply:
> That being said, `emitSkipped` _should_ always have a diagnostic issue. The few times it hasn't (e.g. #338) has been due to a bug, so I'm not sure that there's any semantic difference in it being conditional or unconditional -- the condition is an invariant in this case.

So while creating integration tests in #371 I actually wasn't able to trigger `emitSkipped` what-so-ever, per https://github.com/ezolenko/rollup-plugin-typescript2/pull/371#issuecomment-1174339120 😕  

Then I found https://github.com/microsoft/TypeScript/issues/2290 -- a ~7 year old issue -- which suggests that diagnostic issues might actually _not_ cause `emitSkipped` (at least not when `noEmitOnError: false`).

And in my root cause analysis in https://github.com/ezolenko/rollup-plugin-typescript2/issues/264#issuecomment-1168213567 , I found that a `rootDir` issue can cause `emitSkipped` (which is technically an undetected `compilerOptions` diagnostic due to not having knowledge of all files ahead-of-time).

So I think adding back `noErrors = false` unconditionally might actually be the right play, because it's not really clear what `emitSkipped` is triggered by. I did a bunch of searching and couldn't totally figure it out even from the compiler source code, so I filed https://github.com/microsoft/TypeScript/issues/49790 to help figure out how `emitSkipped` actually works, and listed my current interpretation / hypotheses there.

Reply:
Due to the above, made `noErrors = false` unconditional again in `emitSkipped` in #381